src/ApplicationInsights.ServiceFabric.Native/Net45/IWrappingClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.ServiceFabric.Remoting.Activities
+{
+    using Microsoft.ServiceFabric.Services.Remoting.Client;
+
+    /// <summary>
+    /// Interface implemented by remoting clients to provide access to the inner client.
+    /// </summary>
+    internal interface IWrappingClient
```

Comment:
I'm sorry, I don't get it. 

It seems you can just remove this interface. `CorrelatingServiceRemotingClient` receives `IServiceRemotingClient` in ctor and you can just add IServiceRemotingClient InnerClient field in the  `CorrelatingServiceRemotingClient` that is a typical design pattern.
It really complicates things for the future as you need to check what this wrapper does and think twice to understand why it is needed,